Extension/ui/settings.html

```diff
@@ -423,6 +423,10 @@
 
         <span data-loc-id="switch.to.json">Switch to the <a href="command:C_Cpp.ConfigurationEditJSON" title="Edit configurations in JSON file" data-loc-id-title="edit.configurations.in.json">c_cpp_properties.json</a> file by clicking on the file link or using the command:</span><br> 
         <code data-loc-id="edit.configurations.json">C/C++: Edit configurations (JSON)</code>
+
+        <div style="height: 30px; display: block"></div>
+
+        <span data-loc-id="check.the.schema">Check out <a href="https://code.visualstudio.com/docs/cpp/c-cpp-properties-schema-reference" title="Properties Schema Reference" >C/C++ Properties Schema Reference</a>. Click the link to learn more about the various properties.</span><br>

```

Comment:
@Colengms @michelleangela Any opinions on the wording. I was thinking it would be better to be shorter, like "See the C/C++ Properties Reference to learn more about the various properties.".

Reply:
Okk will shorten it😊

Reply:
How about "Check out the C/C++ properties reference **here**"

@sean-mcmanus

Reply:
Please also make sure to use the localization tag for "title" in the href element.

Reply:
"Check out" seems too informal to be used in UI text. Also, is the usage of "here" necessary if the link is the "C/C++ properties reference"?

Reply:
Please change the wording to the following:

Learn more about the C/C++ properties by going to [C/C++ Properties Schema Reference](https://code.visualstudio.com/docs/cpp/c-cpp-properties-schema-reference).

Reply:
@michelleangela That seems fine to me.

Reply:
Okayy